Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/PerAxisHandles.cs

```diff
@@ -0,0 +1,309 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControlTypes;
+using UnityEngine;
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControl
+{
+    /// <summary>
+    /// Rotation handles for <see cref="BoundsControl"/> that are used for rotating the
+    /// Gameobject BoundsControl is attached to with near or far interaction
+    /// </summary>
+    public abstract class PerAxisHandles : HandlesBase
+    {
+        protected override HandlesBaseConfiguration BaseConfig => config;
+        protected PerAxisHandlesConfiguration config;
+        protected Vector3[] HandlePositions { get; private set; }
+
+        protected abstract int NumHandles { get; }
```

Comment:
Especially critical in the doccomments is how various fields relate (i.e should NumHandles  == HandlePositions size?)